server/build_event_protocol/build_event_server/build_event_server.go

```diff
@@ -16,6 +18,10 @@ import (
 	pepb "github.com/buildbuddy-io/buildbuddy/proto/publish_build_event"
 )
 
+var (
```

Comment:
Instead of introducing a flag, can the sidecar pass an additional option to `NewBuildEventProtocolServer` to enable synchronous proxying?

Reply:
Done.